app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title><%= content_for?(:title) ? yield(:title) : "Untitled" %></title>
+
+    <%= stylesheet_link_tag    "application" %>
+    <%= javascript_include_tag "application", 'data-turbolinks-track' => true %>
+    <%= csrf_meta_tags %>
+  </head>
+
+
+  <body>
+    <header>
+      <h2><%= link_to("Rankster", root_path)%>
+        <small>Ranking the Best of Everything</small>
+      </h2>
+    </header>
+    <nav>
+      <%= link_to("Back to media ranks", root_path, class: "button") %>
+      <%= link_to("View all media", works_path, class: "button") %>
+      <%= link_to("Add new work", new_work_path, class: "button") %>
+      <%= link_to("View all users", users_path, class: "button") %>
+
+  <% if session[:user_id] %>
+    <% @user = User.find(session[:user_id])%>
+    <%= link_to("Logged in as #{@user.name}", user_path(@user.id), class: "button") %>
+    <%= link_to("Logout", logout_path, class: "button", method: :delete) %>
+  <% else %>
+      <%= link_to("Login", login_path, class: "button") %>
+  <% end %>
+    </nav>
+
+    <section class="flash">
+      <% flash.each do |name, message| %>
+      <div class="<%= name %>"><%= message %></div>
+      <% end %>
+    </section>
+    
+    <%= yield %>
+
```

Comment:
As a general rule, it's good to wrap the `yield` in a `<main>` tag, since that's where the page's main content will appear.